__tests__/index.test.js

```diff
@@ -340,6 +340,40 @@ describe('GET /api/articles/:article_id (comment_count)',()=>{
 
 })
 
+describe('GET /api/articles (topic query',()=>{
+    test('return a 200 code when a request is made to the api',()=>{
+        return request(app)
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+    })
+    test('return a 200 code and allarticles filtered by query when a valid request is made to the api "mitch"',()=>{
+
+        return request(app)
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+        .then((response)=>{
+            expect(response.body.articles.length).toBe(12)
+        })
+    })
+    test('return a 200 code and all articles filtered by query when a valid request is made to the api "cats"',()=>{
+        return request(app)
+        .get('/api/articles?topic=cats')
+        .expect(200)
+        .then((response)=>{
+            expect(response.body.articles.length).toBe(1)
+        })
+    })
+    test('return a 200 code and all articles filtered by query when a valid request is made to the api "paper"',()=>{
+        return request(app)
```

Comment:
I'd change the description to reflect the behaviour that is being asserted here (a status 200 - no articles associated with the query)